src/commonMethods/commonFunctions.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
I like the idea of the commonFunctions, but I dont know if src/ in the Test Cases is the best place do to this.
I think this might be better suited in the Storm core. What do you think @michielvandergeest ?

Reply:
Yes I agree. CommonFunctions is a good starting point for reusing code.

But ideally we incorporate functionality that is 'generic' enough to be useful across many into Storm core. Let me come up with an implementation for this feature in Storm core in the next day or so.